analog-weather-glow/usr/share/asteroid-launcher/watchfaces/analog-weather-glow.qml

```diff
@@ -0,0 +1,1156 @@
+    onHrmSensorActiveChanged: {
+        hrmArc.requestPaint()
+        hrmSwitchArc.requestPaint()
+    }
+
+    MceBatteryState {
+        id: batteryChargeState
+    }
+
+    MceBatteryLevel {
+        id: batteryChargePercentage
+    }
+
+    Item {
+        id: dockMode
+
+        readonly property bool active: nightstand
+        property int batteryPercentChanged: batteryChargePercentage.percent
+
+        anchors.fill: parent
+        visible: dockMode.active
+        layer {
+            enabled: true
+            samples: 4
+            smooth: true
+            textureSize: Qt.size(dockMode.width * 2, dockMode.height * 2)
+        }
+
+        Shape {
+            id: chargeArc
+
+            property real angle: batteryChargePercentage.percent * 360 / 100
+            // radius of arc is scalefactor * height or width
+            property real arcStrokeWidth: 0.016
+            property real scalefactor: 0.39 - (arcStrokeWidth / 2)
+            property var chargecolor: Math.floor(batteryChargePercentage.percent / 33.35)
+            readonly property var colorArray: [ "red", "yellow", Qt.rgba(0.318, 1, 0.051, 0.9)]
+
+            anchors.fill: parent
+
+            ShapePath {
+                fillColor: "transparent"
+                strokeColor: chargeArc.colorArray[chargeArc.chargecolor]
+                strokeWidth: parent.height * chargeArc.arcStrokeWidth
+                capStyle: ShapePath.RoundCap
+                joinStyle: ShapePath.MiterJoin
+                startX: width / 2
+                startY: height * ( 0.5 - chargeArc.scalefactor)
+
+                PathAngleArc {
+                    centerX: parent.width / 2
+                    centerY: parent.height / 2
+                    radiusX: chargeArc.scalefactor * parent.width
+                    radiusY: chargeArc.scalefactor * parent.height
+                    startAngle: -90
+                    sweepAngle: chargeArc.angle
+                    moveToStart: false
+                }
+            }
+        }
+
+        Text {
+            id: batteryDockPercent
+
+            anchors {
+                centerIn: parent
+                verticalCenterOffset: parent.width * 0.22
+            }
+            font {
+                pixelSize: parent.width * .15
+                family: "Noto Sans"
+                styleName: "Condensed Light"
+            }
+            visible: dockMode.active
+            color: chargeArc.colorArray[chargeArc.chargecolor]
+            style: Text.Outline; styleColor: "#80000000"
+            text: batteryChargePercentage.percent
+        }
+    }
+
+    Repeater {
+            model: 60
+
+            Rectangle {
+                id: minuteStrokes
+
+                property real rotM: (index - 15) / 60
+                property real centerX: root.width / 2 - width / 2
+                property real centerY: root.height / 2 - height / 2
+
+                x: centerX+Math.cos(rotM * 2 * Math.PI) * parent.width * .46
```

Comment:
Inconsistency alert. You surround mathematical operators with spaces almost everywhere expect this line and the one below.